website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,33 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Select
+
+Multi-select enables users to select multiple rows from a dataset, significantly enhancing the efficiency of batch operations like delete, move, or update. This functionality is important for effective data management, especially in large datasets.
```

Comment:
Maybe a complex answer here, but I think we need to differentiate between rows, tables, results, and datasets. These have different contextual meanings that relate to each other, for example:

- A data set consists of many results
- A table consists of many rows
- A data set can be rendered within a table, the table is the UI representation of the data set. but it can also be rendered in other ways.

So, in this scenario I would maybe approach from this angle:

- "Multi-select enables users to select multiple rows in a table…" or
- "Multi-select enables users to select multiple results from a data set when rendered as a table…"

Not perfect, but I think there's a benefit in the clarity of terminology here.

Reply:
Also, global note: in the filtering docs I intentionally used "data set" rather than "dataset". We should strive for consistency here.

Reply:
To summarize, maybe consider this:

> Multi-select enables users to select multiple rows from a table, allowing for batch actions like delete or update.

The point about effective data management seems a bit broad/ambiguous, is there a way to more accurately express the end-user value?